NEWS.md

```diff
@@ -1,358 +1,315 @@
+  #> │ --- ┆ --- │
+  #> │ f64 ┆ str │
+  #> ╞═════╪═════╡
+  #> │ 1.0 ┆ a   │
+  #> └─────┴─────┘
+  ```
+
+  Use `as_polars_df()` and `as_polars_lf()` to convert existing R `data.frame`s
+  to their `polars` equivalents.
+
+* The class names `PTime` and `rpolars_raw_list` (used to handle time and binary
+  variables) are removed. One should use the classes provided in packages
+  `hms` and `blob` instead.
+
+  ```r
+  ### OLD
+  r_df <- tibble::tibble(
+    time = hms::as_hms(c("12:00:00", NA, "14:00:00")),
+    binary = blob::as_blob(c(1L, NA, 2L)),
+  )
+
+  # R to Polars
+  pl_df <- as_polars_df(r_df)
+  pl_df
+  #> shape: (3, 2)
+  #> ┌─────────┬──────────────┐
+  #> │ time    ┆ binary       │
+  #> │ ---     ┆ ---          │
+  #> │ f64     ┆ list[binary] │
+  #> ╞═════════╪══════════════╡
+  #> │ 43200.0 ┆ [b"\x01"]    │
+  #> │ null    ┆ []           │
+  #> │ 50400.0 ┆ [b"\x02"]    │
+  #> └─────────┴──────────────┘
+
+  # Polars to R
+  tibble::as_tibble(pl_df)
+  #> # A tibble: 3 × 2
+  #>    time binary
+  #>   <dbl> <list>
+  #> 1 43200 <rplrs_r_ [1]>
+  #> 2    NA <rplrs_r_ [0]>
+  #> 3 50400 <rplrs_r_ [1]>
+  ```
+
+  ```r
+  ### NEW
+  r_df <- tibble::tibble(
+    time = hms::as_hms(c("12:00:00", NA, "14:00:00")),
+    binary = blob::as_blob(c(1L, NA, 2L)),
+  )
+
+  ## R to Polars
+  pl_df <- as_polars_df(r_df)
+  pl_df
+  #> shape: (3, 2)
+  #> ┌──────────┬─────────┐
+  #> │ time     ┆ binary  │
+  #> │ ---      ┆ ---     │
+  #> │ time     ┆ binary  │
+  #> ╞══════════╪═════════╡
+  #> │ 12:00:00 ┆ b"\x01" │
+  #> │ null     ┆ null    │
+  #> │ 14:00:00 ┆ b"\x02" │
+  #> └──────────┴─────────┘
+
+  ## Polars to R
+  tibble::as_tibble(pl_df)
+  #> # A tibble: 3 × 2
+  #>   time      binary
+  #>   <time>    <blob>
+  #> 1 12:00  <raw 1 B>
+  #> 2    NA         NA
+  #> 3 14:00  <raw 1 B>
+  ```
+
+
+### Other changes
+
+* R objects that convert to a Series of length 1 are now treated like scalar
+  values when converting to `polars` Expr:
+
+  ```r
+  ### OLD
+  series <- pl$Series("foo", 1)
+  pl$DataFrame(bar = 1:2)$with_columns(series)
+  #> [...truncated...]
+  #> Encountered the following error in Rust-Polars:
+  #>     	Series foo, length 1 doesn't match the DataFrame height of 2
+  #>
+  #>     If you want expression: Series[foo] to be broadcasted, ensure it is a
+  #>     scalar (for instance by adding '.first()').
+  ```
+
+  ```r
+  ### NEW
+  series <- pl$Series("foo", 1)
+  pl$DataFrame(bar = 1:2)$with_columns(series)
+  #> shape: (2, 2)
+  #> ┌─────┬─────┐
+  #> │ bar ┆ foo │
+  #> │ --- ┆ --- │
+  #> │ i32 ┆ f64 │
+  #> ╞═════╪═════╡
+  #> │ 1   ┆ 1.0 │
+  #> │ 2   ┆ 1.0 │
+  #> └─────┴─────┘
+  ```
+
+* `map_batches()` still exists but its usage is discouraged. This function is
+  not guaranteed to interact correctly with the streaming engine. To apply
+  functions from external packages or custom functions that cannot be translated
+  to polars syntax, we now recommend converting the data to a `data.frame` and
```

Comment:
```suggestion
  to polars syntax, we now recommend converting the data to an R DataFrame and
```

Reply:
Disagree on this, I don't remember seeing any other packages where this syntax is used.

Reply:
Ok.